src/client/helpers/setupIconLibrary.js

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (C) 2019  SBVKrishna
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, write to the Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
+
+import {
+	faBook, faChartLine, faCircle, faComment, faEnvelope,
+	faInfo, faPenNib, faPlus, faQuestionCircle, faSearch,
+	faSignInAlt, faSignOutAlt, faUniversity, faUser, faUserCircle,
+	faWindowRestore
+} from '@fortawesome/free-solid-svg-icons';
+import {fab} from '@fortawesome/free-brands-svg-icons';
+import {library} from '@fortawesome/fontawesome-svg-core';
+
+
+// Add Icons to FontAwesome library
+library.add(
+	fab, faBook, faCircle, faChartLine, faComment, faEnvelope,
+	faInfo, faPenNib, faPlus, faQuestionCircle, faSearch,
+	faSignInAlt, faSignOutAlt, faUniversity, faUser, faUserCircle,
+	faWindowRestore
+);
```

Comment:
@MonkeyDo subtasks include some icons not present in this library. Should I add instructions for adding icons to this library or it is better to add all those _going-to-be-used_ icons ?

Reply:
I think for merging the subtask PRs, it'll be a lot easier if there are no file conflicts, and therefore let's try to avoid having multiple participants modify this file.

Reply:
Yeah true. Will update the PR with all icons added :)